acceptance/acceptance_test.go

```diff
@@ -1289,31 +1270,84 @@ func testAcceptance(
 								h.AssertEq(t, output, expectedOutput)
 							}
 						})
+
+						when("builder is untrusted", func() {
+							it("uses the 5 phases", func() {
+								var buf bytes.Buffer
+								cmd := subjectPack(
+									"build", repoName,
+									"--builder", builderName, // untrusted by default
+									"-p", filepath.Join("testdata", "mock_app"),
+									"--publish",
+									"--network", "host",
+								)
+
+								cmd.Stdout = &buf
+								cmd.Stderr = &buf
+
+								h.AssertNil(t, cmd.Start())
+
+								go terminateAtStep(t, cmd, &buf, "[detector]")
+								err := cmd.Wait()
+								h.AssertNotNil(t, err)
+
+								h.AssertContains(t, buf.String(), "[detector]")
+							})
+						})
+
+						when("builder is trusted", func() {
```

Comment:
On thinking about this further, maybe we don't need this test now that this is unit tested in build_test.go... I do think we'll want at least one "happy path" that runs and verifies the 5 phases (as all the other tests, since they are run without `--publish` will use the creator).

Reply:
I am going to remove this test for now. I updated the test assertions in the `--publish` case to verify that the 5 phases were used.